lib/spack/spack/build_environment.py

```diff
@@ -335,6 +336,13 @@ def set_build_environment_variables(pkg, env, dirty):
     env.set(SPACK_DEBUG_LOG_ID, pkg.spec.format('${PACKAGE}-${HASH:7}'))
     env.set(SPACK_DEBUG_LOG_DIR, spack.main.spack_working_dir)
 
+    # Find ccache binary and hand it to build environment
+    if spack.config.get('ccache'):
```

Comment:
I think this is causing the latest failing tests - it should be something like `if spack.config.get('config:ccache'):`

Reply:
Fixed!